Basic Rules.txt

```diff
@@ -43,6 +43,14 @@ which dice represents the tens place before you roll. i.e. if one die rolls a
 die which represents your ten's place and a purple die which represents your one's
 place, and you roll an orange 3 and a purple 9 then you rolled 39%.
 
+Sometimes circumstances or the occasional feat may prompt you to make a roll with

```

Comment:
Can you add a header to this so we can easily find the rule? Such as:

>==== Rolling with Advantage/Disadvtange====

or

>Rolling with Advantage/Disadvantage:

Reply:
Bleehu Likes this post!